### PR TITLE
ExceptionHandler ready

### PR DESCRIPTION
**Pull request by mb83pl**

### CODE REVIEW

File: src/main/java/com/inqoo/trainingservice/app/customer/CustomerService.java [hunk 1]
```diff
@@ -22,21 +25,18 @@ public Customer saveNewCustomer(Customer customer) {
         if (customerRepository.findByEmailAddress(customer.getEmailAddress()).isPresent()) {
             throw new CustomerIsAlreadyExistsException();
         }
```
Comment (D-Petrus) [inline src/main/java/com/inqoo/trainingservice/app/customer/CustomerService.java:27]: format code line

File: src/main/java/com/inqoo/trainingservice/app/exception/ApiExceptionDetails.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package com.inqoo.trainingservice.app.exception;
+
+import java.time.LocalDateTime;
+
+public class ApiExceptionDetails {
+    private LocalDateTime timestamp;
+    private String message;
+
+    public ApiExceptionDetails(LocalDateTime timestamp, String message) {
```
Comment (D-Petrus) [inline src/main/java/com/inqoo/trainingservice/app/exception/ApiExceptionDetails.java:9]: does it have to be public? Make this class immutable

File: src/main/java/com/inqoo/trainingservice/app/exception/ApiExceptionHandler.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.time.LocalDateTime;
+
+@ControllerAdvice
+public class ApiExceptionHandler {
+
+    @ExceptionHandler(NameAlreadyTakenException.class)
+    ResponseEntity<ApiExceptionDetails> handleNameAlreadyTakenException(NameAlreadyTakenException exception) {
```
Comment (D-Petrus) [inline src/main/java/com/inqoo/trainingservice/app/exception/ApiExceptionHandler.java:23]: please group by http status

File: src/test/java/com/inqoo/trainingservice/app/service/CustomerServiceIT.java [hunk 4]
```diff
@@ -91,9 +87,7 @@ public void shouldNotAddCustomerWhenEmailIsExist() {
 
         //then
         customerService.saveNewCustomer(CustomerConverter.dtoToEntity(customer1));
-        Assertions.assertThrows(CustomerIsAlreadyExistsException.class, () -> {
```
Comment (D-Petrus) [inline src/test/java/com/inqoo/trainingservice/app/service/CustomerServiceIT.java:94]: use static imports pls

File: src/main/java/com/inqoo/trainingservice/app/customer/CustomerService.java (hunk 1 above)
Comment (mb83pl) [reply src/main/java/com/inqoo/trainingservice/app/customer/CustomerService.java:27]: done

File: src/main/java/com/inqoo/trainingservice/app/exception/ApiExceptionDetails.java (hunk 2 above)
Comment (mb83pl) [reply src/main/java/com/inqoo/trainingservice/app/exception/ApiExceptionDetails.java:9]: done

File: src/main/java/com/inqoo/trainingservice/app/exception/ApiExceptionHandler.java (hunk 3 above)
Comment (mb83pl) [reply src/main/java/com/inqoo/trainingservice/app/exception/ApiExceptionHandler.java:23]: done

**Pull request merged**